src/main.zig

```diff
@@ -338,8 +340,7 @@ fn draw(t: *Tetris) void {
     }
     {
         var score_text_buf: [20]u8 = undefined;
-        const len = @intCast(usize, c.sprintf(score_text_buf[0..].ptr, c"%d", t.score));
-        const score_text = score_text_buf[0..len];
+        const score_text = try bufPrint(score_text_buf[0..], "{}", t.score);
```

Comment:
I think this should be `catch unreachable` rather than `try`. We can clearly see the buffer size above is big enough to handle the biggest possible score.